packages/scm/src/browser/style/index.css

```diff
@@ -0,0 +1,295 @@
+    overflow-y: auto;
+    border-top: 1px solid var(--theia-layout-color4);
+    width: 100%;
+    flex-grow: 1;
+}
+
+#theia-scmContainer .warn {
+    background-color: var(--theia-error-color2) !important;
+}
+
+#theia-scmContainer .placeholder {
+    width: 100%;
+}
+
+.theia-scm div:focus .theia-mod-selected{
+    background: var(--theia-accent-color3);
+}
+
+.theia-scm .theia-mod-selected{
+    background: var(--theia-accent-color4);
+}
+
+.theia-scm-main-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+
+.theia-scm-input-message-container {
+    display: flex;
+    flex-direction: column;
+    margin: 0px 0px 7px 0px;
+    max-height: 400px;
+}
+
+.flex-container-center {
+    display: flex;
+    align-items: center;
+}
+
+.theia-scm-input-message-container textarea {
+    line-height: var(--theia-content-line-height);
+    font-size: var(--theia-ui-font-size1);
+    color: var(--theia-ui-font-color1);
+    background: var(--theia-layout-color2);
+    border-style: solid;
+    border-width: 1px;
+    resize: none;
+    box-sizing: border-box;
+    padding-left: 4px;
+}
+
+.theia-scm-input-message {
+    width: 100%;
+}
+
+.theia-scm-input-message-idle {
+    border-color: var(--theia-border-color0);
+}
+
+.theia-scm-input-message-info {
+    border-color: var(--theia-info-color0);
+}
+
+.theia-scm-input-message-success {
+    border-color: var(--theia-success-color0);
+}
+
+.theia-scm-input-message-warning {
+    border-color: var(--theia-warn-color0);
+}
+
+.theia-scm-input-message-error {
+    border-color: var(--theia-error-color0) !important;
+}
+
+.theia-scm-message,
+.theia-scm-input-validation-message {
+    padding: 4px 4px 4px 4px;
+}
+
+.theia-scm-validation-message-idle {
+    background-color: var(--theia-border-color0) !important;
+}
+
+.theia-scm-validation-message-info {
+    background-color: var(--theia-info-color0) !important;
+    color: var(--theia-info-font-color0);
+}
+
+.theia-scm-validation-message-success {
+    background-color: var(--theia-success-color0) !important;
+    color: var(--theia-success-font-color0);
+}
+
+.theia-scm-message-warning,
+.theia-scm-validation-message-warning {
+    background-color: var(--theia-warn-color0) !important;
+    color: var(--theia-warn-font-color0);
+}
+
+.theia-scm-validation-message-error {
+    background-color: var(--theia-error-color0) !important;
+    color: var(--theia-error-font-color0);
+}
+
+.no-select {
+    cursor: default;
+    -webkit-user-select: none;
+    -webkit-touch-callout: none;
+    -khtml-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    -o-user-select: none;
+}
+
+.no-select:focus {
+    outline: none;
+}
+
+.theia-scm .scmItem {
+    margin: 1px 0 1px 3px;
+    padding: 2px;
+    font-size: var(--theia-ui-font-size1);
+    display: flex;
+    justify-content: space-between;
+}
+
+.theia-scm .scmItem .file-icon {
+    display: initial;
+}
+
+.theia-scm .scmItem:hover {
```

Comment:
I see that in vs code the headers are selectable. But I don't see why that is. Maybe to indicate that there is a context menu which is not the case here).

Reply:
This style was taken from Git extension.
> Maybe to indicate that there is a context menu which is not the case here).

The item can have a context menu if it is provided by menu contributions. VsCode Git extension provides such menus.